app/helpers/application_helper.rb

```diff
@@ -26,4 +26,12 @@ def state_label(s)
   def minister_warning?(question, minister)
     !!(question.present? && question.open? && minister.try(:deleted?))
   end
+
+  def show_inactive_link(show_inactive)
```

Comment:
I wonder if there is a better name for this method as it can show a link for inactive action officers, but also for active ones?

Reply:
Renamed to show_link